site/pages/our-work/case-study/ft/index.js

```diff
@@ -30,7 +30,7 @@ const FinancialTimesCaseStudy = ({ contactUsURL }: CaseStudyFinancialTimesProps)
     <div className={styles.header}>
       <div className={styles.header__container}>
         <div className={styles.header__imageContainer}>
-          <img src={headerImage} alt="shopping cart" className={styles.header__image} />
+          <img src={headerImage} alt="meeting" className={styles.header__image} />
```

Comment:
Sounds good, although the linter doesn't like the use of "image" in the alt-text so I'll put "Agile stand up in progress" instead if thats okay.

<img width="518" alt="screen shot 2017-08-07 at 15 59 31" src="https://user-images.githubusercontent.com/2177903/29032491-75fb37dc-7b89-11e7-932f-f613537e93d4.png">